sql/fxa_derived/fxa_amplitude_export_v1/query.sql

```diff
@@ -0,0 +1,138 @@
+WITH base_events
```

Comment:
The relevant key is now avaiable to Airflow in BQ with Jason's work in https://bugzilla.mozilla.org/show_bug.cgi?id=1634179#c15

```suggestion
WITH hmac_key AS (
  SELECT
    AEAD.DECRYPT_BYTES(
      (SELECT keyset FROM `moz-fx-dataops-secrets.airflow_query_keys.fxa_prod`),
      (SELECT ciphertext FROM `moz-fx-data-shared-prod.firefox_accounts_derived.encrypted_keys_v1`), key_id) AS value 
  FROM
    `moz-fx-data-shared-prod.firefox_accounts_derived.encrypted_keys_v1`
  WHERE
    key_id = 'fxa_hmac_prod'),
base_events
```